src/Tasks/Microsoft.NET.Build.Tasks/ResolveRuntimePackAssets.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    public class ResolveRuntimePackAssets : TaskBase
+    {
+        public ITaskItem[] ResolvedRuntimePacks { get; set; }
+
+        [Output]
+        public ITaskItem[] RuntimePackAssets { get; set; }
+
+        protected override void ExecuteCore()
+        {
+            List<TaskItem> runtimePackAssets = new List<TaskItem>();
+
+            foreach (var runtimePack in ResolvedRuntimePacks)
+            {
+                string runtimePackRoot = runtimePack.GetMetadata(MetadataKeys.PackageDirectory);
+                string runtimeIdentifier = runtimePack.GetMetadata(MetadataKeys.RuntimeIdentifier);
+
+                string runtimeAssetsPath = Path.Combine(runtimePackRoot, "runtimes", runtimeIdentifier, "lib", "netcoreapp3.0");
+                string nativeAssetsPath = Path.Combine(runtimePackRoot, "runtimes", runtimeIdentifier, "native");
```

Comment:
I guess the plan is that the real runtime pack won't have TFM-sepcific folders, but I still feel a little uneasy about hard-coded netcoreapp3.0 here. Maybe at least link to a bug that tracks changing this?

Reply:
Also, seems we can't just put things in the root as we have to know what is native and what is lib.

Reply:
And the runtime team has a need to keep crossgen in a tools folder.

Reply:
Perhaps this can also be solved with a manifest of some sort like we have tracked for the targeting pack.